packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -139,7 +139,8 @@ type AWSAccountInfo = {
 
 const PINPOINT_TEST_REGEX = /integtest/;
 const APPSYNC_TEST_REGEX = /integtest/;
-const BUCKET_TEST_REGEX = /test/;
+const BUCKET_TEST_REGEX =
```

Comment:
This should be redundant once existing resources are cleaned up. Can you leave a comment explaining this is temporary and what it should be reverted to?